src/cpu/mod.rs

```diff
@@ -236,6 +236,66 @@ impl Cpu {
         self.set_reg(regop, old_val);
     }
 
+    /*
+     * Handle left rotation and left circular rotation.
+     * If RLC (circular), shift left one bit and rotate around
+     * bit 7 to bit 0. Carry flag still holds value of bit 7 prior
+     * to rotation.
+     * If RL, shift left one bit with bit 7 shifting into the carry
+     * flag and the carry flag shifting into bit 0.
+     */
+    fn shift_left(&mut self, regop: Register8Bit, is_rlc: bool, is_a: bool) {
```

Comment:
IDK about passing is_a, I think we should add a check for regop instead. Bread.

Reply:
this will decrease performance (need to match against all Register8Bit) but we can see if this is an optimization issue later